lib/Sabberworm/CSS/CSSList/CSSList.php

```diff
@@ -110,15 +111,34 @@ private static function parseAtRule(ParserState $oParserState) {
 			$oLocation = URL::parse($oParserState);
 			$oParserState->consumeWhiteSpace();
 			$sMediaQuery = null;
-			if (!$oParserState->comes(';')) {
-				$sMediaQuery = $oParserState->consumeUntil(';');
+			try {
+				if (!$oParserState->comes(';')) {
+					$sMediaQuery = $oParserState->consumeUntil(';');
+				}
+				$oParserState->consume(';');
+			} catch (UnexpectedEOFException $e) {
+				// Save the media query if present
+				$sMediaQuery = '';
+				try {
+					while (!$oParserState->isEnd()) {
+						$sMediaQuery .= $oParserState->consume(1);
+					}
+				} catch (UnexpectedEOFException $e) {}
```

Comment:
Can this exception ever be thrown? I would assume not since you’re checking for `!$oParserState->isEnd()`.